src/tags/control/Taxonomy/Taxonomy.js

```diff
@@ -360,8 +362,10 @@ const Model = types
           self._items = items;
         }
       } catch (err) {
+        const message = messages.ERR_LOADING_HTTP({ attr: 'apiUrl', error: err, url:self.apiurl });
```

Comment:
message here is actually a JSON parsing error because non-200 responses are not throwing any errors, so it's not very readable and doesn't explain actual problem (check 404 and missed auth for example).
suggestion to fix it quickly: add check for fetch result and throw actual error from there.